cirq-core/cirq/sim/mux.py

```diff
@@ -25,6 +25,8 @@
 
 from cirq import circuits, devices, ops, protocols, study, value
 from cirq._doc import document
+from cirq.linalg import transformations
+from cirq.protocols import qid_shape_protocol
```

Comment:
`protocols` is imported above, so you can skip this import and use `protocols.qid_shape()` below.